integration/setup/setup.go

```diff
@@ -46,7 +48,20 @@ func installResources(ctx context.Context, config Config) error {
 				WaitForDeploy: true,
 			},
 		}
-		err = config.AppTest.InstallApps(ctx, apps)
+
+		o := func() error {
+			err = config.AppTest.InstallApps(ctx, apps)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			return nil
+		}
+
+		b := backoff.NewConstant(2*time.Minute, 10*time.Second)
```

Comment:
Is it reasonable to wait for 2 minutes?

Reply:
What about increasing to 5 mins?

Right now the test is flaky so waiting a bit longer rather than having to manually re-trigger is a good trade off IMO.

Reply:
damn, I wrote my comments without refreshing, so all is good :D